pandas/tests/io/formats/test_info.py

```diff
@@ -87,7 +87,7 @@ def test_info_verbose():
     frame.info(verbose=True, buf=buf)
 
     res = buf.getvalue()
-    header = " #    Column  Dtype  \n---   ------  -----  "
+    header = " #     Column  Dtype  \n---    ------  -----  "
```

Comment:
does this test for num columns < 1000 and > 1000?

Reply:
This test was only for 1000 columns. There are other test for smaller number of columns.
Anyway, I decided to add parametrized test checking the spacing for various number of columns (``test_info_verbose_with_counts_spacing``), so that it is more clear how the spacing should work.